api/api.v1.js

```diff
@@ -1,7 +1,9 @@
 const router = require("express").Router();
 const adminrouter = require("../routes/admin.routes");
+const userRouter = require("../routes/user.routes");
 
 // Lists all the resources for the restful api
 router.use("/admin", adminrouter);
+router.use("/user", userRouter);
```

Comment:
This should be "/users" because resources are supposed to be pluralized.

Reply:
end point edited